keepAnimals

```diff
@@ -0,0 +1,7 @@
+def keepAnimals(animal_dic):
```

Comment:
Can you rename `animal_dic` to `words`? Since this dictionary can contain animals and non animals, it is confusing to call it `animal_dic`.